src/runtime_functions.js

```diff
@@ -43,26 +43,7 @@ function sigToWasmTypes(sig) {
   return type;
 }
 
-// Wraps a JS function as a wasm function with a given signature.
-function convertJsFunctionToWasm(func, sig) {
-#if WASM2JS
-  return func;
-#else // WASM2JS
-
-  // If the type reflection proposal is available, use the new
-  // "WebAssembly.Function" constructor.
-  // Otherwise, construct a minimal wasm module importing the JS function and
-  // re-exporting it.
-  if (typeof WebAssembly.Function == "function") {
-    return new WebAssembly.Function(sigToWasmTypes(sig), func);
-  }
-
-  // The module is static, with the exception of the type section, which is
-  // generated based on the signature passed in.
-  var typeSection = [
-    0x01, // count: 1
-    0x60, // form: func
-  ];
+function generate_func_type(sig){
```

Comment:
Use JS naming style which would be `generateFuncType`?  Also space before opening paren.